### ISSUE TITLE
In the breadcrumb make the year selectable

### ISSUE THREAD

**Issue by furilo** [PopulateTools/gobierto#275]
When you are into a budget line, we may make the year in the breadcrumb "selectable" (as in the home page), so you can change year over the same budget line. 

Now if you want to see other year for the same budget line you have to start navigating all over again.

**Comment by ferblape** [comment]
Agreed. Also, right now the links of the year are broken, this issue would fix that.
